pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -167,12 +168,21 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	// Watch all the secrets created by this controller so we can regenerate any that are deleted
 	for _, secretName := range []string{
 		render.TigeraElasticsearchCertSecret, render.TigeraKibanaCertSecret,
-		render.OIDCSecretName, render.DexObjectName} {
+		render.OIDCSecretName, render.DexObjectName, relasticsearch.PublicCertSecret,
+		render.KibanaPublicCertSecret} {
```

Comment:
TigeraElasticsearchCertSecret and TigeraKibanaCertSecret hold the cert/key pair while their public counterparts hold just the cert.

> I think since TigeraElasticsearchCertSecret (and TigeraKibanaCertSecret) are not something we need to let the user set we should not store those in the operator namespace.

Until we phase out BYO certs (@rene-dekker created a task for this I believe), we still need these secrets in the operator namespace.

Reply:
I'm marking this resolved because I've got a new comment on this.